sklearn/hmm.py

```diff
@@ -1266,7 +1266,7 @@ def _accumulate_sufficient_statistics(self, stats, obs, framelogprob,
                     cvnorm = np.copy(norm)
                     shape = np.ones(tmp_gmm.covars_.ndim)
```

Comment:
yeah not really, that's true. I am just creating a PR that removes this file from master ;)